plugins/dup.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: dup
+spec:
+  version: v0.2.0
+  homepage: https://github.com/vash/dup
+  shortDescription: Efficiently duplicate existing k8s resources.
```

Comment:
```suggestion
  shortDescription: Duplicate existing Kubernetes resources
```
this would do better as it is more subjective and to the point but I'll let you decide.

Reply:
Sounds great, completely agree with you!